src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -19,7 +19,6 @@ import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import usePostNotificationListenerHook from './getstream/usePostNotificationListenerHook';
 import useSystemMessage from './chat/useSystemMessage';
 import useUserAuthHook from './auth/useUserAuthHook';
-import DatabaseQueue, {DatabaseOperationLabel} from '../../core/queue/DatabaseQueue';
 import {ANONYMOUS, SIGNED} from './constant';
 import {
   AnonymousPostNotification,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `DatabaseQueue, {DatabaseOperationLabel}` has been removed. If `DatabaseQueue` is not used anywhere else in the code, this change is fine. However, if it's used and not imported, it will cause a reference error.
```diff
- import DatabaseQueue, {DatabaseOperationLabel} from '../../core/queue/DatabaseQueue';
+ import {DatabaseOperationLabel} from '../../core/queue/DatabaseQueue';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->